src/utils/helpers/i18n/__spec__.js

```diff
@@ -52,6 +52,15 @@ describe('I18n Helper', () => {
           expect(Helper.formatDecimal('1000', 0)).toEqual('1,000');
           expect(Helper.formatDecimal('100', 1)).toEqual('100.0');
           expect(Helper.formatDecimal('1234567.894', 3)).toEqual('1,234,567.894');
+          expect(Helper.formatDecimal('123.555555555555555', 15)).toEqual('123.555555555555555');
+          expect(Helper.formatDecimal('-987654321987654321.5555555', 7)).toEqual('-987,654,321,987,654,321.5555555');
+          expect(Helper.formatDecimal('-987654321987654321.5555555', 8)).toEqual('-987,654,321,987,654,321.55555550');
+          expect(Helper.formatDecimal('-987654321987654321.5555555', 6)).toEqual('-987,654,321,987,654,321.555556');
+          expect(Helper.formatDecimal('90071992537.40992', 5)).toEqual('90,071,992,537.40992');
+          expect(Helper.formatDecimal('90071992537.40992', 6)).toEqual('90,071,992,537.409920');
+          expect(Helper.formatDecimal('90071992537.40992', 4)).toEqual('90,071,992,537.4099');
+          expect(Helper.formatDecimal('90071992537.409', 5)).toEqual('90,071,992,537.40900');
+          expect(Helper.formatDecimal('90071992537.409', 15)).toEqual('90,071,992,537.409000000000000');
```

Comment:
These are the test cases that the JIRA ticket says are issues
```suggestion
          expect(Helper.formatDecimal('90071992537.409', 15)).toEqual('90,071,992,537.409000000000000');
          expect(Helper.formatDecimal('987654321987654321.55', 2)).toEqual('987,654,321,987,654,321.55');
          expect(Helper.formatDecimal('90071992547.40992', 5)).toEqual('90,071,992,547.40992');
          expect(Helper.formatDecimal('90071992547.406', 5)).toEqual('90,071,992,547.40600');
```